sycl/source/detail/program_manager/program_manager.hpp

```diff
@@ -42,6 +42,9 @@ namespace sycl {
 class context;
 namespace detail {
 
+static constexpr auto inline ITTProfileEnvVarName = "INTEL_LIBITTNOTIFY64";
+static constexpr uint32_t inline ITTSpecConstId = 0xFF747469;
```

Comment:
Should these be in header?

Reply:
They're used in tests and I'd like to avoid duplication.